.github/workflows/live-links.yaml

```diff
@@ -39,7 +39,7 @@ jobs:
             http://www.adlinktech.com/en/index
             /flags.png
             /flags@2x.png
-            http://*.example.com
+            http:\/\/[a-z0-9]*\.example\.com
```

Comment:
Do we care if it matches https?

Reply:
We don't have an example of a secure link but could add it